nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -176,6 +176,10 @@ SECRET_KEY = os.getenv("NAUTOBOT_SECRET_KEY", "{{ secret_key }}")
 #
 # TIME_ZONE = os.getenv("NAUTOBOT_TIME_ZONE", "UTC")
 
+# Show feedback button on NewUI sidebar menu (defualt: True)
+#
+# FEEDBACK_BUTTON_ENABLED = True
+
```

Comment:
Less than ideal location for this in the template (this is the end of the "Django core settings" section). A more appropriate location would be alphabetically inline in the following "Optional settings specific to Nautobot" section, somewhere around line 311 or so.